web/src/components/Header.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { t } from '../i18n';
+
+const Header = () => {
+  return (
+    <header className="header__slim">
+      <div className="ds-l-container">
+        <div className="ds-l-row">
+          <div className="ds-l-col">
+            <Link to="/" className="site-title">
+              {t('titleBasic')}
+            </Link>
+          </div>
+        </div>
+      </div>
+    </header>
+  );
+}
+
+export default Header;
```

Comment:
It could be! I was thinking that it could at least be the header for the admin/card layout sorts of pages that we've got several of (they all seem to use the same header, anyway).